### PR TITLE
gazebo.launch.py has been renamed to ign_gazebo.launch.py for ignition gazebo

### PR DESCRIPTION
**Pull request by ijnek**
Signed-off-by: Kenji Brameld <kenjibrameld@gmail.com>

### CODE REVIEW

File: open_manipulator_x_description/launch/open_manipulator_x_gazebo.launch.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def generate_launch_description():
 
         IncludeLaunchDescription(
             PythonLaunchDescriptionSource([os.path.join(
-                get_package_share_directory("ros_ign_gazebo"), "launch"), "/gazebo.launch.py"]),
+                get_package_share_directory("ros_ign_gazebo"), "launch"), "/ign_gazebo.launch.py"])
```
Comment (wep21) [inline open_manipulator_x_description/launch/open_manipulator_x_gazebo.launch.py:88]: I guess the latest gazebo package is `ros_gz_sim` and the name of launch is `gz_sim.launch.py`.

Comment (ijnek) [reply open_manipulator_x_description/launch/open_manipulator_x_gazebo.launch.py:88]: @wep21 Thanks for the review!
ros_gz_sim was [released into rolling](https://github.com/ros/rosdistro/pull/34914) once, but was then [removed](https://github.com/ros/rosdistro/pull/34925/files).
ros_ign_gazebo will continue to be available in rolling, until the entry is removed from [rolling/distribution.yaml](https://github.com/ros/rosdistro/blob/d4b431569694ce0f112b5a9217bb527e4300de12/rolling/distribution.yaml#L4313-L4335).
That said, we should move to ros_gz_sim once they release it succesfully.

Comment (wep21) [reply open_manipulator_x_description/launch/open_manipulator_x_gazebo.launch.py:88]: I don't know `ros_gz_sim` was once removed. Thank you for letting me know!

Comment (youtalk) [reply open_manipulator_x_description/launch/open_manipulator_x_gazebo.launch.py:88]: Nice review and check! Thank you.

**Review by youtalk — APPROVED** [review]

**Pull request merged**